future/atomicTransactionComposer_test.go

```diff
@@ -194,6 +194,8 @@ func TestAddMethodCallWithManualForeignArgs(t *testing.T) {
 	require.Equal(t, len(txns[0].Txn.ForeignApps), 2)
 	require.Equal(t, txns[0].Txn.ForeignApps[0], types.AppIndex(1))
 	require.Equal(t, txns[0].Txn.ForeignApps[1], types.AppIndex(2))
+	// verify original params object hasn't changed.
+	require.Equal(t, len(params.ForeignApps), 1)
```

Comment:
nit: might as well do `require.Equal(t, params.ForeignApps, []uint64{1})` to compare the value as well